### PR TITLE
Added domain-less provisioning project to mentoring

### PR DESCRIPTION
**Pull request by dLobatog**
@GregSutcliffe sounds good?

### CODE REVIEW

File: mentoring.md [hunk 1]
```diff
@@ -94,6 +94,15 @@ currently available projects.
   an installer to deploy a new host or maybe a Docker image. Working title for
   this is enc2answers.
 
+### Domain-less provisioning for VMs
+
+* *Status:* Not started
+* *Summary of idea*: Users can't create domains on demand on certain cloud services (AWS, DO, etc). We should fetch existing domains from compute resources, or at least allow creating virtual hosts without one. This would make Foreman more accessible as users don't necessarily want to deal with interfaces setup. We could even have an 'advanced' tick button or something that limits the Host form to the minimal requirements unless.
```
Comment (domcleal) [inline mentoring.md]: stray "unless" at the end of the last sentence?

**Comment by domcleal** [comment]
It'd be worth filing a ticket in Redmine too, since it's bordering on a regular feature request, and then linking between the two in case somebody wants to take up the mini-project.

File: mentoring.md (hunk 1 above)
Comment (GregSutcliffe) [inline mentoring.md]: From the preceding structure, I believe that's supposed to be something like "unless the button is checked"

**Comment by GregSutcliffe** [comment]
+1 to filing an RFE, @eLobato can you update this with the ticket link?

**Comment by dLobatog** [comment]
Updated!

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**